lib/ui/common/components/cobble_tile.dart

```diff
@@ -208,6 +208,27 @@ class CobbleTile extends StatelessWidget {
         trailing: child,
       );
 
+  factory CobbleTile.toggle({
```

Comment:
`CobbleTile.app` isn't used anywhere else except in AlertingApps screen in exactly this case, I think you can safely change it to work like you want. You don't have to create new factory, I think.